src/main/java/managers/DatabaseManager.java

```diff
@@ -0,0 +1,60 @@
+package managers;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+import objects.Database;
+
+public class DatabaseManager {
+
+    public Database fullDatabase;
+
+    public DatabaseManager() {
+        // Initialize Full Database
+        JSONParser jsonParser = new JSONParser();
+        try (FileReader reader = new FileReader("src/main/resources/Database.json"))
+        {
+            //Read JSON file
+            Object obj = jsonParser.parse(reader);
+            this.fullDatabase = new Database((JSONObject) obj);
+        } catch (ParseException | IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Database getDatabase(String key) {
+        if (this.fullDatabase.keyExists(key)) {
+            JSONObject subTree = this.fullDatabase.getInfo(key);
+            return new Database(subTree);
+        }
+        return null;
+    }
+
+    public Database getDatabase(Database database, String key) {
+        if (database.keyExists(key)) {
+            JSONObject subTree = database.getInfo(key);
+            return new Database(subTree);
+        }
+        return null;
+    }
+
+    public String getData(String key) {
+        if (this.fullDatabase.keyExists(key)) {
+            JSONObject item = this.fullDatabase.getInfo(key);
+            return item.toString();
+        }
+        return null;
+    }
+
+    public String getData(Database database, String key) {
```

Comment:
From what I understand of whats happening, this getData function needs you to know where the data you want is stored, which is part of the implementation of the database. This would be fine if its private. 
The same with the other methods above - they all depend on the implementation. 

For example, if one day you decided you wanted to change the key in the database that stores enemies for some reason. Then everywhere in the enemy code, you would have to change the key name so it now refers to the new one. 

All in all, I think this is way too general.

It would be much better to have a method like public EnemyData getEnemyData() or public EnemyData getEnemyData(String enemyName).